.changeset/chilled-needles-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+feat(wrangler): use unenv bulitin dependency resolution
```

Comment:
Some more explanation here would be helpful.
Using unenv's resolution for what? For all dependencies? Just unenv ones? etc.

Reply:
I'll fix in a follow-up PR to avoid having to re-run the tests

Reply:
If you only change the md file then most of the tests are skipped.

Reply:
But perhaps not in a PR that has other changes...